lib/core/src/lib/search-text/search-text-input.component.scss

```diff
@@ -5,15 +5,16 @@
 }
 
 .adf-search-button {
-    left: -13px;
+    left: -20px;
+    top: 16px;
```

Comment:
This will affect any other app using this component let's not change it in this way, if we want to fix it it should be fixed by fixing alignement for the whole component not only by moving an icon

Reply:
<img width="679" alt="image" src="https://github.com/user-attachments/assets/0a6ac782-0b4b-4101-8e76-e69f8ba3a261">
@MichalKinas so it's actually an icon misplaced inside the component, so I think that in other apps using adf it would be visible as broken